include/mbgl/map/map.hpp

```diff
@@ -147,6 +159,7 @@ class Map : private util::noncopyable {
 
     inline const TransformState &getState() const { return state; }
     inline std::chrono::steady_clock::time_point getTime() const { return animationTime; }
+    inline util::ptr<AnnotationManager> getAnnotationManager() const { return annotationManager; }
```

Comment:
It doesn't look like shared ownership should be necessary here; `AnnotationManager&` is around for the lifetime of the Map, so return type can be `AnnotationManager&`.